.github/workflows/continuous-integration.yml

```diff
@@ -4,26 +4,25 @@
 name: Continuous Integration
 
 on:
-  pull_request:
-    branches: [ '*' ]
+    pull_request:
+        branches: ['**']
 
 jobs:
-  build:
+    build:
+        runs-on: ubuntu-latest
 
-    runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                node-version: [18.x]
 
-    strategy:
-      matrix:
-        node-version: [18.x]
-
-    steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ matrix.node-version }}
-    - run: yarn
-    - run: yarn lint
-    - run: yarn prettier --check
-    - run: yarn test
-    - run: yarn build
+        steps:
+            - uses: actions/checkout@v2
```

Comment:
With this can you also update the action/checkout to v3,
I think v2 was being depricated

Reply:
Done.
Also updated actions/setup-node to v3